src/legacy/core_plugins/kibana/common/field_formats/types/percent.test.ts

```diff
@@ -17,26 +17,26 @@
  * under the License.
  */
 
-import expect from '@kbn/expect';
-import { createPercentFormat } from '../percent';
-import { FieldFormat } from '../../../../../../../plugins/data/common/field_formats';
+import { createPercentFormat } from './percent';
 
-const PercentFormat = createPercentFormat(FieldFormat);
+const PercentFormat = createPercentFormat();
 
-describe('PercentFormat', function () {
+describe('PercentFormat', () => {
+  const config: Record<any, any> = {};
```

Comment:
If I'm not wrong config have Record<string, any>type

Reply:
Done